appveyor.yml

```diff
@@ -0,0 +1,28 @@
+version: "{build}"
+platform: x64
+clone_folder: c:\gopath\src\github.com\borg323\lczero-client
+
+environment:
+  GOPATH: c:\gopath;c:\gopath\src\github.com\borg323\lczero-client\
```

Comment:
should the borg323 in this PR say lczero-client instead?

Reply:
Nope, fixing.